contracts/5/SushiswapPool12Goblin.sol

```diff
@@ -0,0 +1,246 @@
+pragma solidity 0.5.16;
```

Comment:
ditto here for all changes in SushiSwapGobin.

Reply:
Added comment